microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -84,46 +84,35 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     using System.Threading.Tasks;
     ```  
     
-## Step 4: Drive WebView2 with Selenium and Microsoft Edge Driver  
+## Step 7: Drive WebView2 with Selenium and Microsoft Edge Driver  
 
 1.  First, create the `EdgeOptions` object, by copying the following code snippet.  
     
     ```csharp
     static void Main(string[] args)
     {
-        // EdgeOptions() requires using OpenQA.Selenium.Edge
-        // Construct EdgeOptions with is_legacy = false and the string "webview2"
-        EdgeOptions edgeOptions = new EdgeOptions(false, "webview2");
+
+        EdgeOptions edgeOptions = new EdgeOptions();
+
     ```  
     
-    The `EdgeOptions` object takes in the following two parameters.  
-    
-    | Parameter | Details |    
-    |:--- |:--- |  
-    | `is_legacy` | Set to `false`, which tells Selenium that you are driving the new Chromium-based Microsoft Edge browser. |  
-    | `"webview2"` | A string that tells Selenium you are driving WebView2. |  
-    
-1.  Next, set `edgeOptions.BinaryLocation` to the file path of your WebView2 project runtime, create a string named `msedgedriverDir` that provides the file path to where you installed [Microsoft Edge Driver][MicrosoftDeveloperMicrosoftEdgeWebDriverDownloads], and create a string named `msedgedriverExe` to store the name of the Microsoft Edge Driver runtime.  By default, the runtime is named `msedgedriver.exe`. Use these two strings to construct the `EdgeDriverService` object as shown below.  Finally, create the `EdgeDriver` object using `EdgeDriverService` and `EdgeOptions`.  
+1.  Next, configure `edgeOptions` to use Chromium and WebView2 by setting the options `UseChromium` and `UseWebView` to true. Then, set `edgeOptions.BinaryLocation` to the file path of your host app binary. Finally, create the `EdgeDriver` object using `EdgeOptions`.  
     
     You may copy and paste the following code underneath `edgeOptions`.  Ensure you specify the correct file paths to your project runtime and the Microsoft Edge Driver runtime on your machine.  
```

Comment:
@jasonstephen15 - the code listing doesn't show "specify the correct file paths to ... the Microsoft Edge Driver runtime on your machine."  ok?